RxSwift/Observables/Observable+Multiple.swift

```diff
@@ -62,6 +62,21 @@ extension ObservableType where E : ObservableConvertibleType {
     }
 }
 
+// switchIfEmpty
+
+extension ObservableType {
+    /**
+     Switches to other sequence if source is empty
+     
+     - seealso: [DefaultIfEmpty operator on reactivex.io](http://reactivex.io/documentation/operators/defaultifempty.html)
+     - parameter other: Other sequence which it switches to if the source is empty
+     - returns: Switched observable sequence if source searqence is empty, otherwise emits element from the source sequence.
+     */
```

Comment:
I think we'll need to improve this. I'll try to check out can we reuse some of the texts from `http://reactivex.io/documentation/operators/defaultifempty.html`. 

Maybe something like:
```
Returns the elements of the specified sequence or `switchTo` sequence if the sequence is empty.
```

It's `parameter switchTo` now. 
Maybe `Observable sequence being returned when source sequence is empty.`

Maybe `- returns: Observable sequence that contains elements from switchTo sequence if source is empty, otherwise returns source sequence elements.`